### ISSUE TITLE
Dashboard QA 

### ISSUE THREAD

**Issue by cyberandy** [insideout10/wordlift-plugin#878]
Please describe the issue here:

- [x] add "powered by WooRank" on the bottom right corner of the treemap with a link to WooRank's homepage (including the following parameters `utm_campaign=dashboard` `utm_source=wordlift`)
![image](https://user-images.githubusercontent.com/837037/49723777-f5c59d80-fc67-11e8-91a3-754fae3ce891.png)

- [ ] fix the styling issue related to the _entity type_ when the tile is selected
![pasted_image_at_2018-12-07__3_44_pm](https://user-images.githubusercontent.com/837037/49724570-e8a9ae00-fc69-11e8-8b65-30cc27454e45.png)

- [ ] add on the top right corner of the treemap an indication of the Google Domain configured for the search rankings 
![image](https://user-images.githubusercontent.com/837037/49724676-2575a500-fc6a-11e8-9661-37376f3f1f97.png)

- [ ] add the links to the pages that rank (in the table) and style the table as follows:
![image](https://user-images.githubusercontent.com/837037/49723972-81d7c500-fc68-11e8-855f-72f92f84646d.png)

- [ ] in the Settings group keywords configuration and Google domain in a separated tab (see below) and revise the label related to the Google domain (we currently have _Country_  and _Please select a country_  below the input; we should have _Google Domain_ and _Select your Google domain_ below the input)
![image](https://user-images.githubusercontent.com/837037/49724060-b9467180-fc68-11e8-8014-2bfec50cd3ee.png)

- [x] in the Settings make sure that I can add multiple keywords at once (see something like https://wordpress.org/plugins/bulk-add-terms/ as reference) 

- [x] remove search keywords from the WordLift menu - the users finds the configuration of the Search Rankings in a dedicated tab in the Setting screen 
![image](https://user-images.githubusercontent.com/837037/49724405-7e910900-fc69-11e8-8a47-7988f628bbc4.png)

- [ ] The Search Keywords taxonomy should be visible when Search Rankings isn't visible.

- [x] In Settings the combination Arabic / Egypt displays a not supported error, check.

**Comment by cyberandy** [comment]
We have one more issue to address here 

- [ ] add a link to the entity page in WordPress from the title of the entity-tile in the treemap. This will led the user to the edit page for that entity. We have now a link on the title of each entity-tile but it resolves in a 404 (ie. `https://wordlift.io/wp-admin/admin-ajax.php?action=wl_locate&uri=http://data.wordlift.io/wl0216/entity/artificial_intelligence`) 

![image](https://user-images.githubusercontent.com/837037/55334227-1f196c00-5499-11e9-97d7-b379266b9b4f.png)
